### PR TITLE
Create amazon-connect.md

### PR DESCRIPTION
**Pull request by edumare77**

### CODE REVIEW

File: pages/v2/05-application-connectors/authentication/amazon-connect.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+To connect with Cyclr, you need an AWS account. If you don't have an existing account, you can [sign up for one](https://aws.amazon.com/).
+
+### Get authentication details
+
+To authenticate your connector, you need the following authentication details:
+*  Access Key ID
+*  Access Secret Key
+*  AWS Region
+  
+You can create an access from your AWS account:
```
Comment (erin-lw) [inline pages/v2/05-application-connectors/authentication/amazon-connect.md:22]:
```suggestion
You can create an Access Key from your AWS account:
```

File: pages/v2/05-application-connectors/authentication/amazon-connect.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+To authenticate your connector, you need the following authentication details:
+*  Access Key ID
+*  Access Secret Key
+*  AWS Region
+  
+You can create an Access Key from your AWS account:
+
+1.  Select **Services** and search for **IAM**.
+2.  Under **Access Management**, select **Users**.
+3.  Create a user with the correct permissions, or select an existing one, and select **Security Credentials**.
```
Comment (erin-lw) [inline pages/v2/05-application-connectors/authentication/amazon-connect.md:26]: Need to specify what the correct permissions are

File: pages/v2/05-application-connectors/authentication/amazon-connect.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+4.  Create an **Access Key**.
```
Comment (erin-lw) [inline pages/v2/05-application-connectors/authentication/amazon-connect.md:27]: What do they need to do to create the access key? For example, is it a button that says 'Create Access Key'?

File: pages/v2/05-application-connectors/authentication/amazon-connect.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+5.  Make a note of the **Access Key ID** and the **Access Secret Key**.
+
+### Create an Amazon Connect instance
+  
+In order to use the Amazon Connect connector with Cyclr, you need to set up a virtual contact center instance. For more information on how to set this instance up, see the [[Amazon Guide](https://docs.aws.amazon.com/connect/latest/adminguide/amazon-connect-instances.html).
+  
+You can also create developer and testing instances. For more information see the guide on how to [Create a dev (Sandbox) or test (QA) instance](https://docs.aws.amazon.com/connect/latest/adminguide/create-connect-instance.html).
+  
+* [Find your Amazon Connect instance ID/ARN](Find your Amazon Connect instance ID/ARN)
```
Comment (erin-lw) [inline pages/v2/05-application-connectors/authentication/amazon-connect.md:36]: This doesn't actually have a link!

File: pages/v2/05-application-connectors/authentication/amazon-connect.md (hunk 2 above)
Comment (edumare77) [reply pages/v2/05-application-connectors/authentication/amazon-connect.md]: I've updated this set of instructions, the old ones were copy from Amazon S3.md, the new ones doesn't includes "permissions".

File: pages/v2/05-application-connectors/authentication/amazon-connect.md (hunk 3 above)
Comment (edumare77) [reply pages/v2/05-application-connectors/authentication/amazon-connect.md]: I don't have access to this process, I was handed the credentials, I copy/paste Amazon set of instructions.

File: pages/v2/05-application-connectors/authentication/amazon-connect.md (hunk 4 above)
Comment (edumare77) [reply pages/v2/05-application-connectors/authentication/amazon-connect.md]: Link added

**Pull request merged**